File: ex02/ex02_02/ex02_02.js
function main() {
  const address = "1, Main Str., ";
  const region = "West District, ";
  const city = "Taichung";

  const index1 = 3, index2 = 5, index3 = 7;

  console.log("The (" + String(index1+1) + ")-th character of \"" 
            + address + "\": " + address[index1]);
  console.log("The (" + String(index2+1) + ")-th character of \"" 
            + region + "\": " + region[index2]);
  console.log("The (" + String(index3+1) + ")-th character of \"" 
            + city + "\": " + city[index3]);
  return 0;
}

main();

File: ex02/ex02_02/ex02_02.c
#include <stdio.h>

int main() {
  char address[50] = "1, Main Str., ";
  char region[20] = "West District, ";
  char city[10] = "Taichung";
  int index1 = 3, index2 = 5, index3 = 7;
  printf("The (%d)-th character of \"%s\": %c\n", 
    index1+1, address, address[index1]);
  printf("The (%d)-th character of \"%s\": %c\n", 
    index2+1, region, region[index2]);
  printf("The (%d)-th character of \"%s\": %c\n", 
    index3+1, city, city[index3]);
  return 0;
}